examples/resources/aiven_vpc_peering_connection/import.sh

```diff
@@ -1 +1,2 @@
+terraform import aiven_vpc_peering_connection.mypeeringconnection project/vpc_id/peer_cloud_account/peer_vpc
```

Comment:
Nit: Perhaps adding some comment about what use case of each import statement would be nice